README-source.md

```diff
@@ -587,42 +587,27 @@ This object holds the user's auth details and the data for the API requests.
 
 ```javascript
 {
-  createdBy: '{{chef_name}}'
-  style: '{{style}}'
+  createdBy: 'his name is {{123__chef_name}}'
+  style: 'he cooks {{456__style}}'
 }
 ```
 
+> "curlies" are data mapped in from previous steps. They take the form `{{NODE_ID__key_name}}`. You'll usually want to use `bundle.inputData` instead.
+
 ### `bundle.meta`
 
-`bundle.meta` is extra information useful for doing advanced behaviors depending on what the user is doing. It looks something like this:
+`bundle.meta` is extra information useful for doing advanced behaviors depending on what the user is doing. It has the following options:
 
-```javascript
-{
-  frontend: false,
-  prefill: false,
-  hydrate: true,
-  test_poll: false,
-  standard_poll: true,
-  first_poll: false,
-  limit: -1,
-  page: 0,
-}
-```
-
-For example - if you want to do pagination - you could do:
-
-```javascript
-const getList = (z, bundle) => {
-  const promise = z.request({
-    url: 'http://example.com/api/list.json',
-    params: {
-      limit: 100,
-      offset: 100 * bundle.meta.page
-    }
-  });
-  return promise.then((response) => response.json);
-};
-```
+| key | default | description |
+| ------------- | ------- | ---------------------------------------- |
+| frontend | `false` | whether this poll was initiated via the zap editor |
+| prefill | `false` | whether this poll is being used as a dynamic dropdown |
+| hydrate | `true`  | whether the results of this poll will be hydrated |
+| test_poll | `false` | whether the poll was triggered by a user testing their account |
+| standard_poll| `true`  | the opposite of `test_poll` |
+| first_poll | `false` | whether the poll was an initial check for items when a zap is turned on (none of which will trigger) |
```

Comment:
Whew, this is a tough one to summarize. Maybe we drop what's in the parens and put a link that explains deduplication? So that way when you read it, it will be "Whether the poll was an initial check for items when a Zap turns on. Read more about De-duplication."?

I think something needs to indicate that the items returned populate our de-dupe system.

Reply:
good call, added a dedup section.